modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractScalaCodegen.java

```diff
@@ -57,6 +58,10 @@ public void processOpts() {
         if (additionalProperties.containsKey(CodegenConstants.SOURCE_FOLDER)) {
             this.setSourceFolder((String) additionalProperties.get(CodegenConstants.SOURCE_FOLDER));
         }
+        if (additionalProperties.containsKey(CodegenConstants.STRIP_PACKAGE_NAME) &&
+                "false".equalsIgnoreCase(additionalProperties.get(CodegenConstants.STRIP_PACKAGE_NAME).toString())) {
+            this.stripPackageName = false;
```

Comment:
This should probably also set `additionalProperties` with the boolean value of `false` so it can be used in templates. Otherwise, it will remain `"false"` and this can be confusing in templates. There are helpers in the super class.